tests/cli/test_eval_model.py

```diff
@@ -45,13 +45,16 @@ def test_eval_cli(monkeypatch, tmp_path, capsys):
     assert Path("output.xyz").is_file()
 
 
-def test_eval(monkeypatch, tmp_path, caplog, model, options):
+@pytest.mark.parametrize("model_name", ["model-32-bit.pt", "model-64-bit.pt"])
```

Comment:
In general there is a question to be answered here. Do we eval in the dtype of the exported model or do we allow the user to set the dtype from outside? Can also be left for a future discussion or PR

Reply:
As mentioned above and discussed in slack, the idea would be to only eval in the model's dtype.

Reply:
we can re-open the discussion in slack since both you and @PicoCentauri got a different impression!

Reply:
I think this is basically a decision we have to do. Technically it is easy.

Reply:
> As mentioned above and discussed in slack, the idea would be to only eval in the model's dtype.

Also, this is what we are doing. We use the dtype of the model when loading the structures and the targets.

The test here only checks if we actually can evaluate 64 and 32-bit models. Since we are not using a global dtype anymore and the default dtype is 32-bit we might run into issues when evaluating a 64-bit model.